kotest-assertions/src/jvmMain/kotlin/io/kotest/matchers/equality/reflection.kt

```diff
@@ -190,14 +184,14 @@ fun <T : Any> beEqualToIgnoringFields(
    override fun test(value: T): MatcherResult {
 
       val fieldNames = fields.map { it.name }
-      val failed = value::class.memberProperties.filterNot { fieldNames.contains(it.name) }.mapNotNull {
-         val actual = it.getter.call(value)
-         val expected = it.getter.call(other)
-         if (actual == expected) null else {
-            "${it.name}: $actual != $expected"
+      val fieldsToBeConsider: List<KProperty<*>> = value::class.memberProperties
```

Comment:
Nit: The variable should probably be called "fieldsToBeConsider*d*".

Reply:
Done, renamed to fieldsToBeConsidered